index.ts

```diff
@@ -21,7 +22,7 @@ createSession()
     var server = restify.createServer(serverOptions);
 
     // Setup request logging
-    server.pre((req: restify.Request, res: restify.Response, next: restify.Next): any => {
+    server.pre(function pre(req: restify.Request, res: restify.Response, next: restify.Next): any {
```

Comment:
The restify trace log all middleware function names. The anonymous function will show up as ? in the log, so I decide to give it a name.